tools/ansible/tasks/wsu/main.yaml

```diff
@@ -30,10 +30,8 @@
         - name: Build WMCB
           make:
             target: build
-            chdir: "{{ project_root }}"
-
-        - name: Move WMCB to temporary directory
-          command: mv "{{ project_root }}/{{ wmcb_exe }}" "{{ tmp_dir.path }}/{{ wmcb_exe }}"
```

Comment:
Couldn't you fix it by copying the file instead?

Reply:
We could, but that will leave a copy of wmcb.exe in the project root directory.

Reply:
Is that a problem? Space is cheap :wink:

Reply:
Addressed this